### ISSUE TITLE
HTML template do not create basic HTML tags!

### ISSUE THREAD

**Issue by abc2000x2000** [Komodo/KomodoEdit#3398]
komodo edit 11.0.1

Problem:
HTML template does not create basic HTML tags!

in version 10 when we use
menu > file > new > file from template > html 5 
it creates a html file by this content:
```
<!DOCTYPE html>

<html>
<head>
    <title>Page Title</title>
</head>

<body>



</body>
</html>
```
but version 11.0.1 only show a blank page!
also other htmls this bug exist.

**Comment by Naatan** [comment]
I cannot reproduce this. Did you modify the HTML template in your toolbox?

**Comment by abc2000x2000** [comment]
Hi
No, i did not modify anything.
Html in template in v10 has tags and in v11.0.1 only show blank page.

**Comment by Naatan** [comment]
Can you reproduce this using `Help > Troubleshooting > Restart in Safe-mode` ?

Please verify that when restarting in safe-mode Komodo should be using the default UI (all your settings would be temporarily reset).

**Comment by abc2000x2000** [comment]
Hi
Unfortunately. problem is exist.
I reset to safe mode and use your advise but it did not repair.
version 10 is ok but this version is not OK.

**Comment by Naatan** [comment]
Please share your error log after reproducing the issue.

You can find your error log at Help > Troubleshooting > View Log File.

Also, what OS are you using?

**Comment by DanieleMi** [comment]
Hi all,
I have the same problem, only with versione 11.0.1! 
Version 10.2.3 works perfectly.  My O.S. is Windows 10

**Comment by Naatan** [comment]
@DanieleMi please share your error log after reproducing the issue.

**Comment by abc2000x2000** [comment]
Hi
my OS = windows 10 1703 enterprise
 log:
[2017-12-09 14:18:15,835] [INFO] Startup: Welcome to Komodo Edit 11.0.1
build 18119 (platform win32-x86, running on Windows post2012Server version
6.3.9600)
[2017-12-09 14:18:15,835] [INFO] Startup: D:\Program Files
(x86)\ActiveState Komodo Edit 11\lib\mozilla\komodo.exe built on Fri Oct 27
23:54:50 2017
[2017-12-09 14:18:18,349] [WARNING] console-logger: mutating the
[[Prototype]] of an object will cause your code to run very slowly; instead
create the object with the correct initial [[Prototype]] value using
Object.create (1) in resource://gre/modules/Preferences.jsm:381
[2017-12-09 14:18:18,424] [WARNING] root: [object Object].logging has been
converted to a CommonJS module; use require("ko/logging") instead (since
Komodo 9.0.0a1).
    @chrome://komodo/content/komodo.js:48:1

[2017-12-09 14:18:18,747] [WARNING] console-logger: Expected identifier for
pseudo-class or pseudo-element but found ' '.  Ruleset ignored due to bad
selector. (1) in chrome://komodo/skin/codeintel.css:1
[2017-12-09 14:18:18,749] [WARNING] console-logger: Unexpected end of file
while searching for closing } of invalid rule set. (1) in
chrome://komodo/skin/codeintel.css:42
[2017-12-09 14:18:20,282] [ERROR] console-logger: bad script XDR magic
number (0) in :0
Traceback (most recent call last):
  File "", line 0, in

[2017-12-09 14:18:20,701] [WARNING] keybindings: [F1] was used for
'cmd_helpHelp', overriding to use 'cmd_toggleInteractiveShell'
[2017-12-09 14:18:20,704] [WARNING] keybindings: [Ctrl+0] was used for
'cmd_goToQuickBookmark_0', overriding to use 'cmd_fontZoomReset'
[2017-12-09 14:18:22,193] [INFO] koInitService: Adding pre startup service
for 'koFileStatusService': '@activestate.com/koFileStatusService;1'
[2017-12-09 14:18:22,371] [INFO] elastic_tabstops: Elastic tabstops loaded.
[2017-12-09 14:18:24,486] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_4'
[2017-12-09 14:18:24,486] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_5'
[2017-12-09 14:18:24,486] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_2'
[2017-12-09 14:18:24,486] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_3'
[2017-12-09 14:18:24,487] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_1'
[2017-12-09 14:18:24,489] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_change'
[2017-12-09 14:18:24,489] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'spelling_error'
[2017-12-09 14:18:24,490] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_local_change'
[2017-12-09 14:18:25,191] [INFO] koInitService: Adding pre startup service
for 'KoMemoryReporter': '@activestate.com/koMemoryReporter;1'
[2017-12-09 14:18:25,197] [INFO] koInitService: Adding pre startup service
for 'koCommandmentService': '@activestate.com/koCommandmentService;1'
[2017-12-09 14:18:25,348] [WARNING] console-logger: mutating the
[[Prototype]] of an object will cause your code to run very slowly; instead
create the object with the correct initial [[Prototype]] value using
Object.create (1) in
file:///D:/Program%20Files%20(x86)/ActiveState%20Komodo%20Edit%2011/lib/mozilla/components/koamAddonManager.js:138
[2017-12-09 14:18:33,592] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_4'
[2017-12-09 14:18:33,592] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_5'
[2017-12-09 14:18:33,592] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_2'
[2017-12-09 14:18:33,592] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_3'
[2017-12-09 14:18:33,592] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_1'
[2017-12-09 14:18:33,594] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_change'
[2017-12-09 14:18:33,594] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'spelling_error'
[2017-12-09 14:18:33,594] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_local_change'
[2017-12-09 14:18:45,453] [WARNING] console-logger: Error in parsing value
for 'opacity'.  Declaration dropped. (1) in
chrome://komodo/skin/images/toolbox/command.svg?size=14:0
[2017-12-09 14:18:50,611] [WARNING] views: Inconsistent view count in
view_closed event
[2017-12-09 14:18:50,648] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_4'
[2017-12-09 14:18:50,648] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_5'
[2017-12-09 14:18:50,648] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_2'
[2017-12-09 14:18:50,648] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_3'
[2017-12-09 14:18:50,650] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_1'
[2017-12-09 14:18:50,650] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_change'
[2017-12-09 14:18:50,650] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'spelling_error'
[2017-12-09 14:18:50,651] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_local_change'
[2017-12-09 14:18:50,717] [ERROR] CT::margin.js: couldn't get the
insert-color
-- EXCEPTION START --
TypeError: this.view.scheme.getInterfaceStyleRaw is not a function
+ stack
    exports.MarginController.prototype._initMarkerStyles
@chrome://trackchanges/content/margin.js:59:32
    refreshMarginProperies@chrome://trackchanges/content/margin.js:120:9
    MarginController@chrome://trackchanges/content/margin.js:44:5
    exports.ChangeTracker.prototype.changeTrackingOn
@chrome://trackchanges/content/tracker.js:63:19
    ChangeTracker@chrome://trackchanges/content/tracker.js:27:5
    this.onViewOpenedHandler
@chrome://trackchanges/content/trackchanges.js:88:26
    xtk.domutils.fireEvent@chrome://xtk/content/domutils.js:187:5
    set_koDoc@vieweditor:297:17
    initWithDocument@vieweditor:410:13
    createViewFromDocument@viewlistbase:133:17
    viewManager.prototype._doNewView@chrome://komodo/content/views.js:267:16

viewManager.prototype.doNewViewAsync/<@chrome://komodo/content/views.js:286:20
-- EXCEPTION END --
[2017-12-09 14:18:50,730] [ERROR] CT::margin.js: couldn't get the
delete-color
-- EXCEPTION START --
TypeError: this.view.scheme.getInterfaceStyleRaw is not a function
+ stack
    exports.MarginController.prototype._initMarkerStyles
@chrome://trackchanges/content/margin.js:65:32
    refreshMarginProperies@chrome://trackchanges/content/margin.js:120:9
    MarginController@chrome://trackchanges/content/margin.js:44:5
    exports.ChangeTracker.prototype.changeTrackingOn
@chrome://trackchanges/content/tracker.js:63:19
    ChangeTracker@chrome://trackchanges/content/tracker.js:27:5
    this.onViewOpenedHandler
@chrome://trackchanges/content/trackchanges.js:88:26
    xtk.domutils.fireEvent@chrome://xtk/content/domutils.js:187:5
    set_koDoc@vieweditor:297:17
    initWithDocument@vieweditor:410:13
    createViewFromDocument@viewlistbase:133:17
    viewManager.prototype._doNewView@chrome://komodo/content/views.js:267:16

viewManager.prototype.doNewViewAsync/<@chrome://komodo/content/views.js:286:20
-- EXCEPTION END --
[2017-12-09 14:18:50,737] [ERROR] CT::margin.js: couldn't get the
change-color
-- EXCEPTION START --
TypeError: this.view.scheme.getInterfaceStyleRaw is not a function
+ stack
    exports.MarginController.prototype._initMarkerStyles
@chrome://trackchanges/content/margin.js:71:33
    refreshMarginProperies@chrome://trackchanges/content/margin.js:120:9
    MarginController@chrome://trackchanges/content/margin.js:44:5
    exports.ChangeTracker.prototype.changeTrackingOn
@chrome://trackchanges/content/tracker.js:63:19
    ChangeTracker@chrome://trackchanges/content/tracker.js:27:5
    this.onViewOpenedHandler
@chrome://trackchanges/content/trackchanges.js:88:26
    xtk.domutils.fireEvent@chrome://xtk/content/domutils.js:187:5
    set_koDoc@vieweditor:297:17
    initWithDocument@vieweditor:410:13
    createViewFromDocument@viewlistbase:133:17
    viewManager.prototype._doNewView@chrome://komodo/content/views.js:267:16

viewManager.prototype.doNewViewAsync/<@chrome://komodo/content/views.js:286:20
-- EXCEPTION END --
[2017-12-09 14:18:50,805] [WARNING] views: Inconsistent view count in
view_opened event
[2017-12-09 14:18:50,869] [ERROR] console-logger: bad script XDR magic
number (0) in :0
Traceback (most recent call last):
  File "", line 0, in

[2017-12-09 14:18:50,911] [ERROR] console-logger: TypeError: callback is
not a function (2) in chrome://komodo/content/project/peTemplate.js:77
Traceback (most recent call last):
  File "chrome://komodo/content/project/peTemplate.js", line 77, in

[2017-12-09 14:19:06,563] [WARNING] views: Inconsistent view count in
view_closed event
[2017-12-09 14:19:06,594] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_4'
[2017-12-09 14:19:06,594] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_5'
[2017-12-09 14:19:06,594] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_2'
[2017-12-09 14:19:06,594] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_3'
[2017-12-09 14:19:06,595] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_1'
[2017-12-09 14:19:06,595] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_change'
[2017-12-09 14:19:06,595] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'spelling_error'
[2017-12-09 14:19:06,596] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_local_change'
[2017-12-09 14:19:06,648] [ERROR] CT::margin.js: couldn't get the
insert-color
-- EXCEPTION START --
TypeError: this.view.scheme.getInterfaceStyleRaw is not a function
+ stack
    exports.MarginController.prototype._initMarkerStyles
@chrome://trackchanges/content/margin.js:59:32
    refreshMarginProperies@chrome://trackchanges/content/margin.js:120:9
    MarginController@chrome://trackchanges/content/margin.js:44:5
    exports.ChangeTracker.prototype.changeTrackingOn
@chrome://trackchanges/content/tracker.js:63:19
    ChangeTracker@chrome://trackchanges/content/tracker.js:27:5
    this.onViewOpenedHandler
@chrome://trackchanges/content/trackchanges.js:88:26
    xtk.domutils.fireEvent@chrome://xtk/content/domutils.js:187:5
    set_koDoc@vieweditor:297:17
    initWithDocument@vieweditor:410:13
    createViewFromDocument@viewlistbase:133:17
    viewManager.prototype._doNewView@chrome://komodo/content/views.js:267:16

viewManager.prototype.doNewViewAsync/<@chrome://komodo/content/views.js:286:20
-- EXCEPTION END --
[2017-12-09 14:19:06,657] [ERROR] CT::margin.js: couldn't get the
delete-color
-- EXCEPTION START --
TypeError: this.view.scheme.getInterfaceStyleRaw is not a function
+ stack
    exports.MarginController.prototype._initMarkerStyles
@chrome://trackchanges/content/margin.js:65:32
    refreshMarginProperies@chrome://trackchanges/content/margin.js:120:9
    MarginController@chrome://trackchanges/content/margin.js:44:5
    exports.ChangeTracker.prototype.changeTrackingOn
@chrome://trackchanges/content/tracker.js:63:19
    ChangeTracker@chrome://trackchanges/content/tracker.js:27:5
    this.onViewOpenedHandler
@chrome://trackchanges/content/trackchanges.js:88:26
    xtk.domutils.fireEvent@chrome://xtk/content/domutils.js:187:5
    set_koDoc@vieweditor:297:17
    initWithDocument@vieweditor:410:13
    createViewFromDocument@viewlistbase:133:17
    viewManager.prototype._doNewView@chrome://komodo/content/views.js:267:16

viewManager.prototype.doNewViewAsync/<@chrome://komodo/content/views.js:286:20
-- EXCEPTION END --
[2017-12-09 14:19:06,667] [ERROR] CT::margin.js: couldn't get the
change-color
-- EXCEPTION START --
TypeError: this.view.scheme.getInterfaceStyleRaw is not a function
+ stack
    exports.MarginController.prototype._initMarkerStyles
@chrome://trackchanges/content/margin.js:71:33
    refreshMarginProperies@chrome://trackchanges/content/margin.js:120:9
    MarginController@chrome://trackchanges/content/margin.js:44:5
    exports.ChangeTracker.prototype.changeTrackingOn
@chrome://trackchanges/content/tracker.js:63:19
    ChangeTracker@chrome://trackchanges/content/tracker.js:27:5
    this.onViewOpenedHandler
@chrome://trackchanges/content/trackchanges.js:88:26
    xtk.domutils.fireEvent@chrome://xtk/content/domutils.js:187:5
    set_koDoc@vieweditor:297:17
    initWithDocument@vieweditor:410:13
    createViewFromDocument@viewlistbase:133:17
    viewManager.prototype._doNewView@chrome://komodo/content/views.js:267:16

viewManager.prototype.doNewViewAsync/<@chrome://komodo/content/views.js:286:20
-- EXCEPTION END --
[2017-12-09 14:19:06,707] [WARNING] views: Inconsistent view count in
view_opened event
[2017-12-09 14:19:06,782] [ERROR] console-logger: TypeError: callback is
not a function (2) in chrome://komodo/content/project/peTemplate.js:77
Traceback (most recent call last):
  File "chrome://komodo/content/project/peTemplate.js", line 77, in

[2017-12-09 14:19:17,253] [WARNING] views: Inconsistent view count in
view_closed event
[2017-12-09 14:19:17,283] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_4'
[2017-12-09 14:19:17,283] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_5'
[2017-12-09 14:19:17,283] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_2'
[2017-12-09 14:19:17,283] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_3'
[2017-12-09 14:19:17,285] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_1'
[2017-12-09 14:19:17,286] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_change'
[2017-12-09 14:19:17,286] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'spelling_error'
[2017-12-09 14:19:17,286] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_local_change'
[2017-12-09 14:19:17,342] [ERROR] CT::margin.js: couldn't get the
insert-color
-- EXCEPTION START --
TypeError: this.view.scheme.getInterfaceStyleRaw is not a function
+ stack
    exports.MarginController.prototype._initMarkerStyles
@chrome://trackchanges/content/margin.js:59:32
    refreshMarginProperies@chrome://trackchanges/content/margin.js:120:9
    MarginController@chrome://trackchanges/content/margin.js:44:5
    exports.ChangeTracker.prototype.changeTrackingOn
@chrome://trackchanges/content/tracker.js:63:19
    ChangeTracker@chrome://trackchanges/content/tracker.js:27:5
    this.onViewOpenedHandler
@chrome://trackchanges/content/trackchanges.js:88:26
    xtk.domutils.fireEvent@chrome://xtk/content/domutils.js:187:5
    set_koDoc@vieweditor:297:17
    initWithDocument@vieweditor:410:13
    createViewFromDocument@viewlistbase:133:17
    viewManager.prototype._doNewView@chrome://komodo/content/views.js:267:16

viewManager.prototype.doNewViewAsync/<@chrome://komodo/content/views.js:286:20
-- EXCEPTION END --
[2017-12-09 14:19:17,351] [ERROR] CT::margin.js: couldn't get the
delete-color
-- EXCEPTION START --
TypeError: this.view.scheme.getInterfaceStyleRaw is not a function
+ stack
    exports.MarginController.prototype._initMarkerStyles
@chrome://trackchanges/content/margin.js:65:32
    refreshMarginProperies@chrome://trackchanges/content/margin.js:120:9
    MarginController@chrome://trackchanges/content/margin.js:44:5
    exports.ChangeTracker.prototype.changeTrackingOn
@chrome://trackchanges/content/tracker.js:63:19
    ChangeTracker@chrome://trackchanges/content/tracker.js:27:5
    this.onViewOpenedHandler
@chrome://trackchanges/content/trackchanges.js:88:26
    xtk.domutils.fireEvent@chrome://xtk/content/domutils.js:187:5
    set_koDoc@vieweditor:297:17
    initWithDocument@vieweditor:410:13
    createViewFromDocument@viewlistbase:133:17
    viewManager.prototype._doNewView@chrome://komodo/content/views.js:267:16

viewManager.prototype.doNewViewAsync/<@chrome://komodo/content/views.js:286:20
-- EXCEPTION END --
[2017-12-09 14:19:17,361] [ERROR] CT::margin.js: couldn't get the
change-color
-- EXCEPTION START --
TypeError: this.view.scheme.getInterfaceStyleRaw is not a function
+ stack
    exports.MarginController.prototype._initMarkerStyles
@chrome://trackchanges/content/margin.js:71:33
    refreshMarginProperies@chrome://trackchanges/content/margin.js:120:9
    MarginController@chrome://trackchanges/content/margin.js:44:5
    exports.ChangeTracker.prototype.changeTrackingOn
@chrome://trackchanges/content/tracker.js:63:19
    ChangeTracker@chrome://trackchanges/content/tracker.js:27:5
    this.onViewOpenedHandler
@chrome://trackchanges/content/trackchanges.js:88:26
    xtk.domutils.fireEvent@chrome://xtk/content/domutils.js:187:5
    set_koDoc@vieweditor:297:17
    initWithDocument@vieweditor:410:13
    createViewFromDocument@viewlistbase:133:17
    viewManager.prototype._doNewView@chrome://komodo/content/views.js:267:16

viewManager.prototype.doNewViewAsync/<@chrome://komodo/content/views.js:286:20
-- EXCEPTION END --
[2017-12-09 14:19:17,398] [WARNING] views: Inconsistent view count in
view_opened event
[2017-12-09 14:19:17,482] [ERROR] console-logger: TypeError: callback is
not a function (2) in chrome://komodo/content/project/peTemplate.js:77
Traceback (most recent call last):
  File "chrome://komodo/content/project/peTemplate.js", line 77, in

[2017-12-09 14:19:25,150] [WARNING] views: Inconsistent view count in
view_closed event
[2017-12-09 14:19:25,180] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_4'
[2017-12-09 14:19:25,180] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_5'
[2017-12-09 14:19:25,180] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_2'
[2017-12-09 14:19:25,180] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_3'
[2017-12-09 14:19:25,181] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_1'
[2017-12-09 14:19:25,183] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_change'
[2017-12-09 14:19:25,183] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'spelling_error'
[2017-12-09 14:19:25,183] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_local_change'
[2017-12-09 14:19:25,236] [ERROR] CT::margin.js: couldn't get the
insert-color
-- EXCEPTION START --
TypeError: this.view.scheme.getInterfaceStyleRaw is not a function
+ stack
    exports.MarginController.prototype._initMarkerStyles
@chrome://trackchanges/content/margin.js:59:32
    refreshMarginProperies@chrome://trackchanges/content/margin.js:120:9
    MarginController@chrome://trackchanges/content/margin.js:44:5
    exports.ChangeTracker.prototype.changeTrackingOn
@chrome://trackchanges/content/tracker.js:63:19
    ChangeTracker@chrome://trackchanges/content/tracker.js:27:5
    this.onViewOpenedHandler
@chrome://trackchanges/content/trackchanges.js:88:26
    xtk.domutils.fireEvent@chrome://xtk/content/domutils.js:187:5
    set_koDoc@vieweditor:297:17
    initWithDocument@vieweditor:410:13
    createViewFromDocument@viewlistbase:133:17
    viewManager.prototype._doNewView@chrome://komodo/content/views.js:267:16

viewManager.prototype.doNewViewAsync/<@chrome://komodo/content/views.js:286:20
-- EXCEPTION END --
[2017-12-09 14:19:25,244] [ERROR] CT::margin.js: couldn't get the
delete-color
-- EXCEPTION START --
TypeError: this.view.scheme.getInterfaceStyleRaw is not a function
+ stack
    exports.MarginController.prototype._initMarkerStyles
@chrome://trackchanges/content/margin.js:65:32
    refreshMarginProperies@chrome://trackchanges/content/margin.js:120:9
    MarginController@chrome://trackchanges/content/margin.js:44:5
    exports.ChangeTracker.prototype.changeTrackingOn
@chrome://trackchanges/content/tracker.js:63:19
    ChangeTracker@chrome://trackchanges/content/tracker.js:27:5
    this.onViewOpenedHandler
@chrome://trackchanges/content/trackchanges.js:88:26
    xtk.domutils.fireEvent@chrome://xtk/content/domutils.js:187:5
    set_koDoc@vieweditor:297:17
    initWithDocument@vieweditor:410:13
    createViewFromDocument@viewlistbase:133:17
    viewManager.prototype._doNewView@chrome://komodo/content/views.js:267:16

viewManager.prototype.doNewViewAsync/<@chrome://komodo/content/views.js:286:20
-- EXCEPTION END --
[2017-12-09 14:19:25,253] [ERROR] CT::margin.js: couldn't get the
change-color
-- EXCEPTION START --
TypeError: this.view.scheme.getInterfaceStyleRaw is not a function
+ stack
    exports.MarginController.prototype._initMarkerStyles
@chrome://trackchanges/content/margin.js:71:33
    refreshMarginProperies@chrome://trackchanges/content/margin.js:120:9
    MarginController@chrome://trackchanges/content/margin.js:44:5
    exports.ChangeTracker.prototype.changeTrackingOn
@chrome://trackchanges/content/tracker.js:63:19
    ChangeTracker@chrome://trackchanges/content/tracker.js:27:5
    this.onViewOpenedHandler
@chrome://trackchanges/content/trackchanges.js:88:26
    xtk.domutils.fireEvent@chrome://xtk/content/domutils.js:187:5
    set_koDoc@vieweditor:297:17
    initWithDocument@vieweditor:410:13
    createViewFromDocument@viewlistbase:133:17
    viewManager.prototype._doNewView@chrome://komodo/content/views.js:267:16

viewManager.prototype.doNewViewAsync/<@chrome://komodo/content/views.js:286:20
-- EXCEPTION END --
[2017-12-09 14:19:25,289] [WARNING] views: Inconsistent view count in
view_opened event
[2017-12-09 14:19:25,364] [ERROR] console-logger: TypeError: callback is
not a function (2) in chrome://komodo/content/project/peTemplate.js:77
Traceback (most recent call last):
  File "chrome://komodo/content/project/peTemplate.js", line 77, in

[2017-12-09 14:19:35,095] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_4'
[2017-12-09 14:19:35,095] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_5'
[2017-12-09 14:19:35,095] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_2'
[2017-12-09 14:19:35,095] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_3'
[2017-12-09 14:19:35,096] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_cursor_1'
[2017-12-09 14:19:35,096] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_remote_change'
[2017-12-09 14:19:35,098] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'spelling_error'
[2017-12-09 14:19:35,098] [WARNING] koScintillaSchemeService: applyScheme::
no indicator for name 'collab_local_change'
[2017-12-09 14:19:59,434] [WARNING] console-logger: Key event not available
on GTK2: key="u" modifiers="control,shift" (1) in
chrome://komodo/content/tail/tail.xul:0

**Comment by Naatan** [comment]
Thanks @abc2000x2000, that's helpful.

**Comment by abc2000x2000** [comment]
Yes after you say about toolbox i found that and add code to templates.
Thanks for helping.

**Comment by cgchoffman** [comment]
I can't think of a reasonable upgrade path here.  If the user has this issue and they use templates then they have edited them and therefore we should not over right them.

To get the default File Templates again you have to do a profile reset (save your current profile) and migrate the generated File Templates to your your current profile.

**Comment by squidgerton** [comment]
@cgchoffman -- Following b/c I'm having the same problem.  I just downloaded 11 (Mac OS) and haven't edited or modified templates. 

However, the default templates, all of them, are completely blank.

**Comment by cgchoffman** [comment]
The fix will be in 11.1 @squidgerton.

**Comment by abc2000x2000** [comment]
thanks
